themes/bootstrap3/templates/RecordDriver/DefaultRecord/explain-line.phtml

```diff
@@ -3,53 +3,38 @@
     <?=$this->localizedNumber($explainElement['percent'], $decimalPlaces)?>%
   </td>
   <td>
-    <?php if (is_array($fieldName)): ?>
-      Synonym: [
-      <?php for ($i = 0; $i < count($fieldName); $i++): ?>
-        <?php
-        $name = $fieldName[$i];
-        $value = $fieldValue[$i] ?? '';
-        $exactMatch = $explainElement['exactMatch'][$i] ?? '';
-        if (!empty($name)) {
-          $description = $this->transEsc('IndexFieldDescription::' . $name);
-          $description = $name !== $description ? $description : '';
-        } else {
-          $description = '';
-        }
-        $exactMatchClass = ($exactMatch ?? 'unkown') . '-match';
-        ?>
-        <span class="<?=$exactMatchClass?>">"<?=$value?>"</span>
-        <?=$this->transEsc('in')?> <span class="field-name" title="<?=$this->escapeHtmlAttr($description)?>"><?=$name?></span><?=($i < count($fieldName) - 1) ? ', ' : ''?>
-      <?php endfor; ?>
-      <?php
-      $modifier = isset($explainElement['fieldModifier'])
-        ? ', ' . $this->transEsc(
-            'explain_modifier',
-            ['%%modifier%%' => $this->localizedNumber($explainElement['fieldModifier'], $decimalPlaces)]
-        )
-        : '';
-      ?>
-      ]<?=$modifier?>
-    <?php else: ?>
-      <?php
-      if (!empty($fieldName)) {
-        $fieldDescription = $this->transEsc('IndexFieldDescription::' . $fieldName);
-        $fieldDescription = $fieldName !== $fieldDescription ? ' (' . $fieldDescription . ')' : '';
+    <?php if (!is_array($fieldName)): ?>
```

Comment:
The indentation and not using curly brackets for pure-PHP blocks makes this a bit difficult to read. Also, could we just make it so that everything is always an array so that the template doesn't need to check these?

Reply:
Refactored to always use arrays and changed the block in the template to pure-PHP with curly brackets.